<launch>
	<node name="$(anon traj_vector_server)" pkg="traj_splitter_to_vector" type="traj_vector_server" cwd="node" output="screen" />
	<node name="$(anon walking_trajectory_vector)" pkg="traj_splitter_to_vector" type="walking_trajectory_vector" cwd="node" output="screen" />
</launch>
